samples/bootloader/README.rst

```diff
@@ -69,6 +69,11 @@ By default, the bootloader sample will automatically generate and provision publ
 Alternatively, to facilitate the manufacturing process of a device with the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
 If you choose to do so, use the Python scripts in ``scripts\bootloader`` to create and provision the keys manually.
 
+   .. note::
```

Comment:
This line shouldn't be indented (and the next lines accordingly - three spaces).

Reply:
This indentation scheme is used elsewhere in this document, e.g. https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1818/files/3dd334ab8b6c24fdffa16477fd864eae1f92d1af#diff-0f7f2103603f132c693d35086cfa585bR106

Should I fix them?

Reply:
No - there it is part of a step.